src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import './variables';
+@import './card';
+@import './stars';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
```

Comment:
Consider moving general styles to a separate file to keep the main file clean and clear